.github/workflows/windows-vcpkg.yml

```diff
@@ -0,0 +1,37 @@
+name: Windows vcpkg Workflow
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  schedule:
+  # Execute a "nightly" build at 2 AM UTC 
+  - cron:  '0 2 * * *'
+
+jobs:
+  build:
+    name: 'Windows vcpkg job'
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    
+    # Restore from cache the previously built ports. If "cache miss"
+    # then provision vcpkg, install desired ports, finally cache everything for the next run.
+    - name: Restore from cache and run vcpkg
+      uses: lukka/run-vcpkg@v3
+      with:
+        vcpkgArguments: '--triplet x64-windows poco'
+        # commit from vcpkg's master branch on 2020/06/01
+        vcpkgGitCommitId: 6d36e2a86baf8d227fc6dce587bd69997d67fb5e
+        vcpkgDirectory: '${{ github.workspace }}/vcpkg'
```

Comment:
Should this be `vcpkgDirectory: '${{ github.workspace }}/../vcpkg'` instead?

I am not especially knowledgeable in this subject, I am just comparing with https://github.com/ros-industrial/abb_libegm/pull/101#discussion_r433370614

Reply:
Yes, indeed the error is due to that. Note that once we will be able to link the imported CMake targets for the dependency and have a proper relocatable  package, we will not have this limitation.

Reply:
@traversaro: I'm confused, as in your comment on `abb_libegm` you seemed to imply this was only a problem because of the Protobuf stuff there.

But from your comment here it seems to be due to the fact we're not using `target_link_libraries(..)` with Poco yet.

Reply:
Sorry, probably I was not clear. You have that error whenever you call  `target_include_directories(exported_target PUBLIC <directory/that/is/contained/inside/source/dir>)`, and it is meant to be an early error to avoid that somebody by mistake exports a CMake target that has an include that points to the source directory headers instead of the directory of the installer headers. We are doing `target_include_directories(exported_target PUBLIC <directory/that/is/contained/inside/source/dir>)` both in the `abb_libegm` repo (because we don't use protobuf imported  targets, and we include the protobuf  header directories "manually" via `target_include_directories(... PUBLIC ${Protobuf_INCLUDE_DIRS})`) and also in this repo, because we don't use poco imported  targets, and we include the poco  header directories "manually" via `target_include_directories(... PUBLIC ${Poco_INCLUDE_DIRS})`

Reply:
Ah, clear. You've confirmed what I suspected.